akka-stream/src/main/scala/akka/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -581,7 +581,7 @@ final class GraphStageIsland(
   private val logicArrayType = Array.empty[GraphStageLogic]
   private[this] val logics = new ArrayList[GraphStageLogic](64)
```

Comment:
This is an ArrayList so no explicit resize logic needs to be added. I will change it to have the same initial size, as it seems to be used in similar places as `connections` is.